tests/framework/Optimizers/stochastic.xml

```diff
@@ -4,19 +4,20 @@
     <name>framework/Optimizers/Stochastic</name>
```

Comment:
the name should be equal to the one that is used to run the test

eg. ```<name>framework/Optimizers.Stochastic</name>``` because I can run the case using the following command: ```./run_tests --re=framework/Optimizers.Stochastic```